src/system/pooling.js

```diff
@@ -172,6 +172,18 @@
             // store back the object instance for later recycling
             entityClass[name].pool.push(obj);
         };
+        
+        /**
+         * Check if an object with the provided name is registered
+         * @name exists
+         * @memberOf me.pool
+         * @public
+         * @function
+         * @param {String} name of the registered object
```

Comment:
that's the input parameter, you're using it this way in your other functions too

https://github.com/xorinzor/melonJS/blob/master/src/system/pooling.js#L59